src/libsync/propagatedownloadzsync.cpp

```diff
@@ -177,12 +179,10 @@ void GETFileZsyncJob::seedFinished(void* zs)
     }
     _propagator->reportFileTotal(*_item, zsyncSize);
 
-    _zs.release();
+    _zs = zsp.release();
```

Comment:
Couldn't `_zs` *be* the unique_ptr?

Reply:
Yes and `_zr` and maybe `_zbyterange` ... looking into it now. `std::move` is how to allocate right?

Reply:
you can do `_zr = std::unique_ptr...;` to create it. you'd need to `move` only to transfer ownership somewhere else.

Reply:
Done, for `_zs`, `_zr` and `_zbyterange`.